bcda/api/alr.go

```diff
@@ -97,11 +92,14 @@ func (h *Handler) alrRequest(w http.ResponseWriter, r *http.Request, reqType ser
 	// Use a transaction to guarantee that the job only gets created if we queue all of the alrJobs
 	rtx := postgres.NewRepositoryTx(tx)
 
-	alrJobs, err := h.Svc.GetAlrJobs(ctx, ad.CMSID, req, service.AlrRequestWindow{LowerBound: rp.Since})
+	alrMBIs, err := h.r.GetAlrMBIs(ctx, ad.CMSID)
```

Comment:
ALR does not take a time period window.  It fetches all the data associated with the latest ALR entry in the alr_meta table.